zkevm-circuits/src/evm_circuit/util/common_gadget.rs

```diff
@@ -439,3 +445,227 @@ impl<F: Field> TransferGadget<F> {
         Ok(())
     }
 }
+
+#[derive(Clone, Debug)]
+pub(crate) struct CommonCallGadget<F, const IS_SUCCESS_CALL: bool> {
+    pub is_success: Cell<F>,
+
+    pub gas: Word<F>,
+    pub gas_is_u64: IsZeroGadget<F>,
+    pub callee_address: Word<F>,
+    pub value: Word<F>,
+    pub cd_address: MemoryAddressGadget<F>,
+    pub rd_address: MemoryAddressGadget<F>,
+    pub memory_expansion: MemoryExpansionGadget<F, 2, N_BYTES_MEMORY_WORD_SIZE>,
+
+    value_is_zero: IsZeroGadget<F>,
+    pub has_value: Expression<F>,
+    pub callee_code_hash: Cell<F>,
```

Comment:
suggest use old name `phase2_callee_code_hash` so that we easily know clear which phase the RLC do .

Reply:
fixed in [8607418](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1023/commits/86074189be1416a61effd63461a9c938da133f42)